DirectEmployers/MyJobs
# [PD-2673]  change mobile breakpoint to 768px

Purpose of this PR is to gracefully transition from desktop view to iPhone 5 sized devices view.  End users will get the "full Monty" at screen size 992px and greater.  Between 991px and 769px, end users get a larger tablet view with smaller version of the desktop top bar and smaller footer.  At 768px and below, end users will get the full mobile web interface.

To test:

1.  Please switch to v2 template.

2. Navigate in desktop view, not much change here.

3. Decrease browser size to less than 991px but greater than 768px, please ensure that there is a "Filter Search Criteria" button, please click on it, this will toggle the visibility of the filter facets.  Please ensure search functionality still works.

4. Decrease browser size to less than 768px or use Chrome's mobile device simulator tools for a specific mobile device.  Please ensure the mobile view shows up, this is the same mobile view that is already in Quality Control now.
